tests/unit/test_tiler.py

```diff
@@ -672,7 +673,7 @@ def but_with_wrong_coordinates(self, request, tmpdir):
 
     def and_doesnt_raise_error_with_wrong_coordinates(self, request, tmpdir):
```

Comment:
i know this is old ... but please replace the name with : `and_it_does_not_raise_error_with_wrong_coordinates` . In general tests name should be verbose and this name contains some weird thing like `doesnt` that it is not even english :D and the `it` is missing.